src/index.js

```diff
@@ -0,0 +1,8 @@
+import './style.css';
+
+import addTodo from '../modules/addTodo.js';
+import TodoList from '../modules/workClass.js';
+
+const todoclass = new TodoList();
+todoclass.workList();
+addTodo();
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md#:~:text=js%20file%20and-,set%20an%20array%20of%20some%20simple%20to%20do%20tasks%20(array%20of%20objects).%20Each%20task%20object%20should%20contain%20three%20keys,-%3A), Please set an array of some simple to do tasks (array of objects). Each task object should contain three keys:
- description [string].
- completed [bool].
- index: [number].

- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md#:~:text=index%3A%20%5Bnumber%5D.-,Write%20a%20function%20to%20iterate%20over%20the%20tasks%20array%20and%20populate%20an%20HTML%20list%20item%20element%20for%20each%20task.,-On%20page%20load), Kindly Write a function to iterate over the tasks array and populate an HTML list item element for each task.